config/locales/translations/zendesk_apps_support.yml

```diff
@@ -145,17 +145,23 @@ parts:
       title: "App builder job: invalid locations"
       value: "%{invalid_locations} are invalid locations in %{host_name}."
   - translation:
+      key: "txt.apps.admin.error.app_build.blank_location_uri"
+      title: "App builder job: invalid URI for an iframe in the manifest"
+      value: "%{location} location does not specify a URI."
```

Comment:
hello! could you give me an example of complete sentence here? what exactly is loaded in the placeholder, a path?